documents/motions/10.2020.0 - Committees and Teams.md

```diff
@@ -14,6 +14,7 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
 2. All WCA Committees and Teams shall be formed for an indefinite period, unless otherwise decided by the WCA Board of Directors.
 3. Each WCA Committee and Team will have a direct relationship with at least one Director for management, communication, and escalation.
 4. Each WCA Committee and Team shall have one Leader.
+   0. Exception: The WCA Executive Assistants Team is led by the WCA Board.
```

Comment:
Format question: Aren't the Motion numbers are reassigned when inclouding new articles?

Reply:
Yes, I think it's better if this is 4.1 and the next becomes 4.2, etc. I don't think we've used x.0 in other documents.

Reply:
Fixed this in upcoming amendments.